lib/aur-search

```diff
@@ -9,63 +9,51 @@ query_type=search
 search_by=name-desc
 sort_key=Name
 
-json_short() {
-    local Name Version NumVotes Maintainer OutOfDate Description
+parse_short() {
+    jq -r --arg key "$1" '[.results[]] | sort_by(.[$key])[] | [
+        .Name         // "-",
+        .Version      // "-",
+        .NumVotes     // "-",
+        .Maintainer   // "-",
+        .OutOfDate    // "-",
+        .Description  // "-"
+        ] | @tsv'
```

Comment:
unrelated change, no benefit from going tsv instead of newline separated (more compact notation but makes tools like awk and read that use newlines by default harder to use)

Reply:
https://github.com/AladW/aurutils/pull/328#discussion_r176923571

Reply:
This is needed for removing duplicates.